README.md

```diff
@@ -116,3 +116,14 @@ If you just want to execute the test suite, you can use:
 
 > make clean test
 
+## Releasing a new version of `nsq-j` (for maintainers)
+
+### Bump the library version
+The `nsq-j` version is referenced in several places, such as this README,
+as well as in the pom file. See a prior release PR such as [this one](https://github.com/sproutsocial/nsq-j/pull/66)
+for an example.
+
+### Create a tag for the release commit
+
+### Create a release for the new tag
+
```

Comment:
still need to fill these sections out; suggestions appreciated 😄

Reply:
Created a PR into this release branch with my suggested changes: https://github.com/sproutsocial/nsq-j/pull/76/files